### ISSUE TITLE
Intermittent calico start-up failure

### ISSUE THREAD

**Issue by alexbrand** [apprenda/kismatic#29]
The calico-node service fails to start intermittently during our automated test runs.

Full build link [here](https://snap-ci.com/apprenda/kismatic-platform/branch/master/logs/defaultPipeline/317/DEAC-Integration-Test?back_to=build_history)

```
- Running task: calico : verify calico-node is running
  ip-10-0-3-87                                                                  [OK]
  ip-10-0-3-201                                                                 [ERROR]
---- STDOUT ----
● calico-node.service - Calico per-node agent
   Loaded: loaded (/etc/systemd/system/calico-node.service; enabled; vendor preset: disabled)
   Active: activating (auto-restart) (Result: exit-code) since Tue 2016-11-01 13:58:02 UTC; 6s ago
     Docs: https://github.com/projectcalico/calico-docker
  Process: 12749 ExecStart=/usr/bin/calicoctl node --ip=${INTERNAL_IP} --node-image=${DOCKER_IMAGE} --detach=false (code=exited, status=1/FAILURE)
 Main PID: 12749 (code=exited, status=1/FAILURE)

Nov 01 13:58:02 ip-10-0-3-201 systemd[1]: calico-node.service: main process exited, code=exited, status=1/FAILURE
Nov 01 13:58:02 ip-10-0-3-201 systemd[1]: Unit calico-node.service entered failed state.
Nov 01 13:58:02 ip-10-0-3-201 systemd[1]: calico-node.service failed.
---------------
  ip-10-0-3-131                                                                 [OK]
  ip-10-0-3-8                                                                   [OK]
  ip-10-0-3-107                                                                 [ERROR]
---- STDOUT ----
● calico-node.service - Calico per-node agent
   Loaded: loaded (/etc/systemd/system/calico-node.service; enabled; vendor preset: disabled)
   Active: activating (auto-restart) (Result: exit-code) since Tue 2016-11-01 13:58:02 UTC; 6s ago
     Docs: https://github.com/projectcalico/calico-docker
  Process: 12691 ExecStart=/usr/bin/calicoctl node --ip=${INTERNAL_IP} --node-image=${DOCKER_IMAGE} --detach=false (code=exited, status=1/FAILURE)
 Main PID: 12691 (code=exited, status=1/FAILURE)

Nov 01 13:58:02 ip-10-0-3-107 systemd[1]: calico-node.service: main process exited, code=exited, status=1/FAILURE
Nov 01 13:58:02 ip-10-0-3-107 systemd[1]: Unit calico-node.service entered failed state.
Nov 01 13:58:02 ip-10-0-3-107 systemd[1]: calico-node.service failed.
---------------
```

**Comment by alexbrand** [comment]
This should be resolved by updating our calico version to > v0.22

**Comment by dasmb** [comment]
Should be rushed. Time to build calico upgrade playbooks!

**Comment by dkoshkin** [comment]
Should be fixed in #214
